### PR TITLE
Update to SLEAP v1.2.7

### PR DESCRIPTION
**Pull request by roomrys**
### Description
Update references from 1.2.6 to 1.2.7.

### Types of changes

- [ ] Bugfix
- [ ] New feature
- [ ] Refactor / Code style update (no logical changes)
- [x] Build / CI changes
- [ ] Documentation Update
- [x] Other (release)

### Does this address any currently open issues?
[list open issues here]

### Outside contributors checklist

- [ ] Review the [guidelines for contributing](https://github.com/talmolab/sleap/blob/develop/docs/CONTRIBUTING.md) to this repository
- [ ] Read and sign the [CLA](https://github.com/talmolab/sleap/blob/develop/sleap-cla.pdf) and add yourself to the [authors list](https://github.com/talmolab/sleap/blob/develop/AUTHORS)
- [ ] Make sure you are making a pull request against the **develop** branch (not *main*). Also you should start *your branch* off *develop*
- [ ] Add tests that prove your fix is effective or that your feature works
- [ ] Add necessary documentation (if appropriate)

#### Thank you for contributing to SLEAP!
:heart:

### CODE REVIEW

**Review by roomrys — COMMENTED** [review]
Builds on both windows and ubuntu locally

File: requirements.txt [hunk 1]
```diff
@@ -14,7 +14,6 @@ pandas
 psutil
 PySide2>=5.13.2,<=5.14.1; platform_machine != 'arm64'
 PySide2>=5.13.2,<=5.15.5; sys_platform == 'darwin' and platform_machine == 'arm64'
-qtpy>=2.0.1
```
Comment (talmo) [inline requirements.txt:17]: why did we remove qtpy from requirements.txt?

Comment (roomrys) [reply requirements.txt:17]: we found a [qtpy conda package](https://anaconda.org/anaconda/qtpy) (>=v2.0.1) that has support for both windows, ubuntu, and mac, so we moved this requirement to the conda environment yamls

File: environment.yml [hunk 2]
```diff
@@ -16,7 +16,8 @@ dependencies:
   - nvidia::cuda-nvcc=11.3
   # - sleap::tensorflow=2.7.0
   # - sleap::pyside2=5.14.1
-  - conda-forge::pip<=22.0.3
+  - qtpy>=2.0.1
```
Comment (roomrys) [inline environment.yml:19]: takes care of the `qtpy` requirement for win/ubuntu with GPU

File: environment_no_cuda.yml [hunk 3]
```diff
@@ -16,7 +16,8 @@ dependencies:
   # - nvidia::cuda-nvcc=11.3
   # - sleap::tensorflow=2.7.0
   # - sleap::pyside2=5.14.1
-  - conda-forge::pip<=22.0.3
+  - qtpy>=2.0.1
```
Comment (roomrys) [inline environment_no_cuda.yml:19]: takes care of the `qtpy` requirement for win/ubuntu w/o GPU

**Pull request merged**